pkg/kubelet/types/labels.go

```diff
@@ -17,25 +17,34 @@ limitations under the License.
 package types
 
 const (
-	KubernetesPodNameLabel       = "io.kubernetes.pod.name"
-	KubernetesPodNamespaceLabel  = "io.kubernetes.pod.namespace"
-	KubernetesPodUIDLabel        = "io.kubernetes.pod.uid"
+	// KubernetesPodNameLabel is the label of Kubernetes pod name
```

Comment:
Specifically, these are labels attached to the container by the container runtime.